en_US.json

```diff
@@ -1109,12 +1114,14 @@
 	"hint.directive.origin_read_timeout": "set origin read timeout",
 	"hint.directive.origin_send_timeout": "set origin send timeout",
 	"hint.directive.origin_set_header": "modify request header to origin",
-	"hint.directive.proxy_buffering": "buffer the response from upstream",
+	"hint.directive.proxy_buffering": "deprecated",
 	"hint.directive.proxy_cache_background_update": "enable refresh the cache in the background",
 	"hint.directive.proxy_cache_bypass": "set the conditions to bypass cache",
-	"hint.directive.proxy_cache_lock": "lock to avoid simultaneous requests to fill cache",
-	"hint.directive.proxy_cache_lock_age": "minimum interval to grant requests to fill cache",
+	"hint.directive.proxy_cache_convert_head": "convert a HEAD request to a GET request",
+	"hint.directive.proxy_cache_lock": "deprecated. cache lock turned on globally",
```

Comment:
Should it be "Cache lock" instead of "cache lock" ?

Reply:
I just used the text suggested by Neil.